coalib/output/ConsoleInteraction.py

```diff
@@ -316,7 +316,11 @@ def print_result(console_printer,
                                                      file_diff_dict,
                                                      section)
                 cli_actions = tuple(action for action in cli_actions
-                                    if not isinstance(action, ShowPatchAction))
+                                    if not (isinstance(action, ShowPatchAction)
+                                            or
+                                            isinstance(action,
+                                                       ShowAppliedPatchesAction)
+                                            ))
```

Comment:
I think you can squeeze this into `isinstance(action,(ShowPatchAction, ShowAppliedPatchesAction))` to make it a bit short 👍

Reply:
Nice suggestion Ishan, but it makes the line longer than 81 characters in all the permutations of , and (.
:wink:

Reply:
```
                                    if not (isinstance(action, (
                                        ShowPatchAction, ShowAppliedPatchesAction))
```
or 
```
                                    if not (isinstance(action, (
                                        ShowPatchAction, 
                                        ShowAppliedPatchesAction))
```
either of this should work.